src/v2/Apps/Home/HomeApp.tsx

```diff
@@ -63,6 +64,8 @@ export const HomeApp: React.FC<HomeAppProps> = ({
         <HomeFeaturedGalleriesRailQueryRenderer />
 
         <HomeTrendingArtistsRailQueryRenderer />
+
+        <HomeTroveArtworksRailQueryRenderer />
```

Comment:
Is this the right place? Shouldn't this be up higher in the list like between works for you and market news? 🤔

Reply:
Idk if there is a specification on where it is supposed to be on the Homepage. 

The design docs show it at the top under the main carousel in both the signed in and signed out flow.